x-pack/test/apm_api_integration/basic/tests/service_overview/transaction_groups.ts

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { pick, uniqBy } from 'lodash';
+import url from 'url';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+import archives from '../../../common/archives_metadata';
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const esArchiver = getService('esArchiver');
+
+  const archiveName = 'apm_8.0.0';
+  const { start, end } = archives[archiveName];
+
+  describe('Service overview transaction groups', () => {
+    describe('when data is not loaded', () => {
+      it('handles the empty state', async () => {
+        const response = await supertest.get(
```

Comment:
nit: This is still typed as `any` but not terribly important.

Reply:
WDYT about a client that wraps `supertest` and automatically gets the type based on the url and method - similar to what we use in APM?

```ts
const res = apmSupertest({
 endpoint: 'GET /api/apm/services/opbeans-java/overview_transaction_groups',
 // ... 
})

Reply:
I like it 👍 I think there's a small chance of us being less inclined to test things that don't align with the types, but generally I would say the benefits outweigh that risk.